obsplus/bank/utils.py

```diff
@@ -70,24 +79,34 @@ def _get_path(info, path, name, path_struct, name_strcut):
     return dict(path=path, filename=out_name)
 
 
-def _summarize_wave_file(path, format):
-    if format == "mseed":
-        try:
-            return summarize_mseed(path)
-        except Exception:
-            pass
-    # specialized mseed function failed
-    out_list = []
+def summarize_generic_stream(path, format=None):
     st = _try_read_stream(path, format=format) or _try_read_stream(path) or []
+    out = []
     for tr in st:
-        out = {
-            "starttime": tr.stats.starttime.timestamp,
-            "endtime": tr.stats.endtime.timestamp,
+        summary = {
+            "starttime": tr.stats.starttime._ns,
```

Comment:
Yes, in the current branch of ObsPy `UTCDateTime` has a public `ns` attribute which returns the same thing, so it will be public soon (hopefully). We just need to remember to update ObsPlus when it happens.

Reply:
Knowing that, I think it is ok to use `_ns` until `ns` is available.